packages/botonic-api/src/rest/routes/events.ts

```diff
@@ -109,26 +102,26 @@ router
           .send({ error: `Event with ID '${params.eventId}' not found` })
         return
       }
+      if (event.eventId !== updatedEvent.eventId) {
+        res
+          .status(400)
+          .send({ error: `Both event ID (params and body) must match` })
+        return
+      }
```

Comment:
This shouldn't be needed. The ID is already encoded in the URL, there should be no ID in the body (or if there is, just ignore it and always use the URL one)

Reply:
Make sense, fixed!